Makefile

```diff
@@ -36,6 +36,13 @@ build:
 build-native:
 	cargo build -p micro-rdk-server --bin micro-rdk-server-native
 
+build-installer:
+	cargo build -p micro-rdk-installer --bin micro-rdk-installer --release
+
+install-tools:
```

Comment:
It sounds like we should remove the `install-tools` target and only retain `build-installer.